include/email.js

```diff
@@ -61,7 +65,7 @@ module.exports = function EmailServiceModule(pb) {
      */
     EmailService.prototype.sendFromTemplate = function(options, cb){
         var self = this;
-        var ts   = new pb.TemplateService();
+        var ts   = new pb.TemplateService({ site: this.site });
```

Comment:
I hate that I wrote it this way.  Need to add a "TODO" to move the instantiation of the template service to the constructor so it can be injectable with all of the other context properties it needs.